src/components/Header.tsx

```diff
@@ -25,6 +25,8 @@ export default function Header() {
         </Link>
       </HeaderButton>
 
+      <Link href="/terms">License</Link>
```

Comment:
```suggestion
      <div className="flex w-fit items-center gap-6">
      	<Link href="/terms">License</Link>
	      <HeaderButton>
	        <a href={siteMetadata.GITHUB_URL} target="_blank">
	          <SiGithub size={24} />
	        </a>
	      </HeaderButton>
      </div>
```

以下のように License を端に寄せた方がメニューっぽくてわかりやすいと思うのですが、どう思いますか？👀

<img width="774" alt="スクリーンショット 2024-09-03 22 33 40" src="https://github.com/user-attachments/assets/c525c119-e2a1-45a9-a7ce-1f939fa45fd7">

Reply:
ヘッダーデザインしては、現状下記のような感じが、簡潔かつ、最も使いやすいかな？と考えてました！
（ヒーロセクションも一緒ですが！）

- license: 利用規約
- contoributing: TOPページ下部の、デザイン追記のコントリビュートセクション
- （各デザインの詳細画面、、？など）

なので、モーダルのデザインのイシューもあるので、
ヘッダーデザインもいったん別でもいいかもなーと思ってます！

いかがでしょう？🙌
<img width="555" alt="スクリーンショット 2024-09-04 4 17 09" src="https://github.com/user-attachments/assets/f29d1746-380c-4885-9065-88ffb2cb8573">

Reply:
ランダム機能って、ランダムに並べ変えるのではなく、ランダムに画像を生成するってこと、ですかね？👀
もしそうだとしたら、最初に決めていたコンセプト（質の高いオリジナルLGTM画像作成、LGTM画像でサービスでチームの雰囲気を良くする）から結構ぶれることになると思うので、自分は無い方が良いかなと思います！👍

ただ、もっと無難なLGTM画像ジェネレータにコンセプトを変更したい、というなら、それでも良いと思います！
任せますー👍

あと、今気付いたのですが、利用規約やプライバシーポリシーなどは普通フッターにあるので、フッターを作って置くのが良いかもですね🤔
ヘッダーにはよく使うメニュー＝機能がある、という認識ですー！それ以外を置くのは、コンテンツの邪魔になるかなと！

Reply:
> モーダルにある、Twitterシェアですが、現状では、個別のデザインをOGで投稿できなそうなので、モーダルにある意味ないかなーとかも思っている感じで、少し悩んでる感じです。。

OGで投稿できるとモーダルの意味があって、OGで投稿できないとモーダルの意味がない、ということでしょうか？
もう少し詳しくお聞かせいただきたいです！🙏

Reply:
あ、色々書いてしまい、少しややこしかったですね！

とりあえず、
モーダル内のデザインのイシューもあるので、
ヘッダーデザインも、いったん別イシューで、いいかもなーと思っているじです！

その他、コメントいただいた、ランダム機能・ヘッダー・Twitterシェアについては、
下記のスレッドに書きますね！
多分規約ページのPRとは少し外れる、アイデア的な感じの提案なので👍
- #184

Reply:
了解です！👍

Reply:
レビュー・提案ありがとうございます！！🙌